config/co2eq_parameters_direct.json

```diff
@@ -3187,9 +3187,9 @@
           }
         ],
         "unknown": {
-            "_url": "https://docs.google.com/spreadsheets/d/1CegROfej9HqRZTfihpjPpgZTYPUHrTNgTdHWbbP3w74/edit#gid=291258352",
```

Comment:
Could we also run formatting on json files?

Reply:
We can run prettier on them. I was planning on adding that and a action for it after #4171 is merged.

Reply:
That would be awesome yeah!